### DISCUSSION TITLE
GitHub Copilot failed to activate in IDEA

### DISCUSSION THREAD

**Discussion by Rela-ted** [community/community#19399] [Copilot Conversations]
Click GitHub Copilot in the tool to activate it. Log in to GitHub account for authorization as prompted. IDEA is displayed, but the IDEA interface is still waiting for the browser to authorize the login account，How can I solve the problem?

**Comment by rohith1125** [comment]
They made the service a paid subscription model now, so it doesn't work, I had the same issue, these are the steps I followed,

1. Uninstall the extension
2. Reinstall the extension
3. Complete the login process

It will stop showing the server error and work :)
